.changeset/famous-timers-mix.md

```diff
@@ -0,0 +1,5 @@
+---
+'@vercel/node': minor
```

Comment:
Any reason you chose minor? Is this adding a new feature?

Reply:
https://github.com/vercel/vercel/pull/10328